sphinx/users/imagej/index.rst

```diff
@@ -1,7 +1,7 @@
 ImageJ overview
 ===============
 
-`ImageJ <https://imagej.nih.gov/ij/index.html>`_ is an image processing and
+`ImageJ <https://imagej.net/>`_ is an image processing and
```

Comment:
```suggestion
`ImageJ <https://imagej.net/ij/index.html>`_ is an image processing and
```
Point to the equivalent of the previously-referenced ImageJ main page, not the one that references Fiji/ImageJ2 since instructions here are not compatible with Fiji.